app/assets/stylesheets/components/_slider.scss

```diff
@@ -0,0 +1,199 @@
+$track-width: 16em;
+$track-height: .625em;
+$track-bg: $white;
+$track-fill: linear-gradient(90deg, $grey-3 25%, $grey-3 100%);
+
+$thumb-diameter: 1.5em;
+
+$option-colours: (
+  "leave-pot-untouched": $color-option-leave-pot-untouched,
+  "guaranteed-income": $color-option-guaranteed-income,
+  "adjustable-income": $color-option-adjustable-income,
+  "take-cash": $color-option-take-cash,
+  "whole-pot": $color-option-take-whole-pot,
```

Comment:
Agree, but this is consistent with the other CSS components so am tempted to keep as is and maybe fix in a separate PR?